templates/zerver/app/index.html

```diff
@@ -3,6 +3,9 @@
 {# The app itself. #}
 {# Includes some other templates as tabs. #}
 
+{% set PAGE_TITLE = "Public view of {} | Zulip".format(realm_name) %}
+{% set PAGE_DESCRIPTION = "Browse the publicly accessible channels in the {} Zulip team chat without logging in.".format(realm_name) %}
+
```

Comment:
Hm, this ended up harder to parse in situ than I was hoping. Maybe something like this?

{% set PAGE_TITLE = "Public view of {} | Zulip team chat".format(realm_name) %}
{% set PAGE_DESCRIPTION = "Browse the publicly accessible channels in {} without logging in.".format(realm_name) %}

@timabbott wdyt?

Reply:
Here's a quick screenshot of the Zulip message with that text change:

![Screenshot from 2024-12-12 13-41-35](https://github.com/user-attachments/assets/a74ccb4d-c116-4dc7-97ce-6eebe2883304)

Reply:
This seems pretty good.